site/viamguides/src/securibot/securibot.md

```diff
@@ -0,0 +1,318 @@
+   ```bash
+   pwd
+   ```
+
+### Configure a vision service
+
+1. In the Viam app, click the **+** icon in the left-hand menu and select **Service**, and then `vision`.
+   <!-- ![select vision](assets/vision.png) -->
+1. Search for a module called `face-identification`. Then click **Add module**, and **Create** a new vision service, renaming the service as `face-identification`.
+   <!-- ![add module](assets/apriltag.png) -->
+1. Notice this creates two new items in the left sidebar. The first is your new vision service called `face-identification` based on a module also called `face-identification`.
+   ![two new items in the sidebar](assets/twoItems.png)
+1. In the `face-identification` panel for the vision service under the **CONFIGURE** section, add the following attributes. This configures the vision service to depend on data coming in from the named webcam, refer to a directory of `known_faces` to be stored on the Raspberry Pi, and establish a sensitivity threshold for the vision service.
+
+   ```json
+   {
+     "picture_directory": "/home/joncha/known_faces",
+     "identification_threshold": 0.5,
+     "camera_name": "camera-1"
+   }
+   ```
+
+   ![configure vision service](assets/configureService.png)
+
+1. **Save** your changes in the top right and wait a few moments for the configuration changes to take effect.
+1. At the bottom of the `face-identification` vision service panel, expand the **TEST** section to ensure you have configured the vision service properly. Point the webcam towards a face to see if the camera detects it.
+1. If the camera detects a face, a bounding box will highlight the item in the video feed, and display it on the right under **Labels**.
+   <!-- ![apriltag test](assets/apriltagDetected.png) -->
+
+### Configure an automation service
+
+Next, let’s set up a service to ensure the camera is always running the facial detection program. When a recognized face is detected, the servo will activate to unlock the door. If the face is not recognized, no action will be taken.
+
+1. In the Viam app, click the **+** icon in the left-hand menu and select **Service**, and then `generic`. Then search for a module called `securibot:doorbot`.
+   ![add module](assets/moduleSecuribot.png)
+1. Click **Add module**, and **Create** a new vision service, renaming the service as `automation`.
+1. Notice this creates two new items in the left sidebar. The first is your new generic service called `automation` based on a module called `doorbot`.
+1. In the `automation` panel for the generic service under the **CONFIGURE** section, add the following attributes. This configures the service to depend on data coming in from the named webcam (`camera-1`), trigger the named servo (`servo-1`), and named vision service (`face-identification`).
+   ```json
+   {
+       "camera_name": <string>,
+       "servo_name": <string>,
+       "vision_name": <string>
```

Comment:
We should fill in the configuration values so it's easy to copy and paste into the app:

```suggestion
       "camera_name": "camera-1",
       "servo_name": "servo-1",
       "vision_name": "face-identification"
```